Neos.Flow/Classes/Core/Bootstrap.php

```diff
@@ -540,11 +540,11 @@ protected function defineConstants()
             define('FLOW_PATH_TEMPORARY', $temporaryDirectoryPath);
         }
 
-        // Not using this flag and loading classes via our class loader is deprecated
-        // and we will remove class loading from our loader in the next major of Flow (5.0).
-        $onlyUseComposerAutoLoaderForPackageClasses = false;
-        if (in_array(self::getEnvironmentConfigurationSetting('FLOW_ONLY_COMPOSER_LOADER'), [true, 'true', 1, '1'])) {
-            $onlyUseComposerAutoLoaderForPackageClasses = true;
+        // Using this flag and loading classes via our class loader is deprecated
+        // and we will remove class loading from our loader in the next major of Flow (8.0).
```

Comment:
Good catch! I missed this one because I was searching for `@deprecated` annotations. I think we should add that.
Also in most places we now specify in which version the code became deprecated instead of trying to foresee when it will be removed.
I think this makes sense. Maybe something like:

```suggestion
        // Setting this flag to false will enable the custom Class Loader on top of the default autoloading provided by composer
        // @deprecated sind Version 4.3. Packages should use the default composer autoloading mechanism
```